src/main/java/biz/netcentric/filevault/validator/aem/cloud/AemCloudValidator.java

```diff
@@ -119,10 +136,14 @@ static boolean isPackagePathInstalledConditionally(String runMode, Path packageR
 
     @Override
     public @Nullable Collection<ValidationMessage> done() {
+        Collection<ValidationMessage> messages = new ArrayList<>();
         if (hasInstallHooks && hasMutableNodes) {
-            return Collections.singleton(new ValidationMessage(defaultSeverity, VIOLATION_MESSAGE_INSTALL_HOOK_IN_MUTABLE_PACKAGE));
+            messages.add(new ValidationMessage(defaultSeverity, VIOLATION_MESSAGE_INSTALL_HOOK_IN_MUTABLE_PACKAGE));
         }
-        return null;
+        if (packageType == null && hasMutableNodes && hasImmutableNodes) {
```

Comment:
Why only considering the case when `packageType == null`? Shouldn't we consider the case when the packageType is "mixed" as well?
Or even any packageType? (because we shouldn't mix mutable and inmutable content in any kind package)

Reply:
For packageType=mixed there is https://github.com/Netcentric/aem-cloud-validator/pull/13/files#diff-40fa06d0a7f2d00656da282b483e91c498e023cef1a898e90ceea7ca36877031R98, i.e. the validation message is emitted immediately.
For other package types the standard validator `jackrabbit-packagetype` is enough (https://jackrabbit.apache.org/filevault/validation.html)